docs/main.unmin.js

```diff
@@ -104,6 +83,18 @@ lib.walk = async (dmap, path) => {
     return {meta, data}
 }
 
+lib.parse = (path) => {
+    if (path === '') return []
+
+    const baned = path.match(/[^a-z.:]/)
+    const names = path.match(/[a-z]+/g)
+    const runes = path.match(/[:.]/g)
+
+    need(names.length === runes.length && baned === null, `Invalid dpath`)
+
```

Comment:
Feel this is unlikely to be correct, what about `:a..b` or similar 
I think ebnf should be one of the last things to try to drop

Reply:
An empty name like ```:a..b``` is fine because then names.length will not equal runes.length (2 != 3), and it'll be rejected.
Picked off ebnf so dmfxyz can look at ethers

Reply:
Good point, it is still hard to convince myself it is correct though, also I think using a grammar is something worth showing, like keeping the dependency intentionally
Will think some more